pge_file_lib_globs.cpp

```diff
@@ -1340,23 +1340,24 @@ void FileInfo::rebuildData()
     m_baseName.clear();
 
     //Take full path
+
 #ifdef PGE_FILES_QT
     m_filePath = QFileInfo(m_filePath).absoluteFilePath();
-#else
-#   ifndef _WIN32
+#elif defined(__3DS__)
+    // all paths are absolute on 3DS
```

Comment:
Please initialize the `m_filePath` with anything like `m_filePath = ".";`, or simply add the `m_filePath.clear();` here to keep it always blank, this field gets never clean as it usually gets written by calls.

Reply:
The purpose of the code I'm skipping is to turn m_filePath, which was already initialized, from a relative path to an absolute path. So I do not think we want to change it here in a system without relative paths. All the other variables are still cleared as they were before.

Reply:
Okay, I did check, you right, I hadn't looked at this code file for a while, and didn't remind it's structure, my fault. Gonna merge this. :fox_face: :+1: